.github/workflows/ci.yml

```diff
@@ -74,7 +74,15 @@ jobs:
         node-version: '16'
     - name: Install Packages
       run: npm install
+      env:
+        DETECT_CHROMEDRIVER_VERSION: true
     - name: Test
-      run: node Makefile karma
+      run: node Makefile wdio
     - name: Fuzz Test
       run: node Makefile fuzz
+    - uses: actions/upload-artifact@v3
+      if: failure()
+      with:
+        name: logs
+        path: |
+          wdio-logs/*.log
```

Comment:
How will we download & see these logs?

Reply:
> How will we download & see these logs?

In the summary of a job under `Artifacts`, see picture below (bottom):
![Screenshot 2023-07-26 at 21 27 09](https://github.com/eslint/eslint/assets/731337/121c745b-edf3-4861-8ee6-48794efbe410)

https://github.com/eslint/eslint/actions/runs/5583652074